src/stylesheets/elements/_typography.scss

```diff
@@ -31,6 +31,99 @@ a {
   }
 }
 
+%usa-heading {
+  clear: both;
+  font-family: $font-serif;
+  line-height: $heading-line-height;
+  margin-bottom: 0.5em;
+  margin-top: 1.5em;
+}
+
+@mixin usa-typography-base-styles {
+  p {
+    @extend %usa-paragraph;
+  }
+
+  a {
+    @extend %usa-link;
+  }
+}
+
+@mixin usa-headings-styles {
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    @extend %usa-heading;
+  }
+
+  h1 {
+    @include h1;
+  }
+
+  h2 {
+    @include h2;
+  }
+
+  h3 {
+    @include h3;
+  }
+
+  h4 {
+    @include h4;
+  }
+
+  h5 {
+    @include h5;
+  }
+
+  h6 {
+    @include h6;
+    font-family: $font-sans;
+  }
+}
+
+// Basic typography styles (paragraph text and links)
+@mixin usa-global-styles-basic {
+  @if $usa-global-styles-basic == true {
+    @include usa-typography-base-styles;
+  }
+}
+
+%usa-content-styles {
+  @include usa-typography-base-styles;
+  @include usa-headings-styles;
+  @include usa-list-styles;
+  @include usa-table-styles;
+}
+
+/* stylelint-disable block-closing-brace-newline-after, at-rule-empty-line-before */
+@mixin usa-content-styles {
+  @if $usa-content-styles == 'scoped' {
+    .usa-prose {
+      @extend %usa-content-styles;
+    }
+  } @else if $usa-content-styles == 'global' {
+    @extend %usa-content-styles;
+  } @else {
+    // Default nothing if it’s neither
+  }
+}
+/* stylelint-enable */
+
+@include usa-global-styles-basic;
```

Comment:
I really like this placeholder --> mixin --> conditional --> include technique. Clean!